sdks/java/io/amazon-web-services/src/main/java/org/apache/beam/sdk/io/aws/options/AwsModule.java

```diff
@@ -132,6 +135,10 @@ public AWSCredentialsProvider deserializeWithType(
         return new ProfileCredentialsProvider();
       } else if (typeName.equals(EC2ContainerCredentialsProviderWrapper.class.getSimpleName())) {
         return new EC2ContainerCredentialsProviderWrapper();
+      } else if (typeName.equals(STSAssumeRoleSessionCredentialsProvider.class.getSimpleName())) {
+        return AssumeRoleSessionCredentialsProvider.getInstance(
```

Comment:
Can we just return `new STSAssumeRoleSessionCredentialsProvider.Builder(asMap.get(ROLE_ARN), asMap.get(ROLE_SESSION_NAME)).build()` here? Why do we need to create `AssumeRoleSessionCredentialsProvider` class?